props_lab/src/components/donation.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react'
+import TopBar from './topBar'
+import Progress from './progress'
+import Form from './form'
+ import DonationReason from './DonationReason'
+
+class Donation extends Component{
+
+    render(){
+        return(
+            <div>
+            <TopBar/>
```

Comment:
You could have done all that in the App.js since all that App.js has is this `<Donation/> ` component